setup.cfg

```diff
@@ -88,21 +88,21 @@ dev =
     %(tests)s
     dvc-data[cli]
 # Extra dependencies for remote integrations
-azure = dvc-azure==0.0.1
-gdrive = dvc-gdrive==0.0.1
-gs = dvc-gs==0.0.1
-hdfs = dvc-hdfs==0.0.1
-oss = dvc-oss==0.0.1
-s3 = dvc-s3==0.0.2
-ssh = dvc-ssh==0.0.1
+azure = dvc-azure==2.19.0
+gdrive = dvc-gdrive==2.19.0
+gs = dvc-gs==2.19.0
+hdfs = dvc-hdfs==2.19.0
+oss = dvc-oss==2.19.0
+s3 = dvc-s3==2.19.0
+ssh = dvc-ssh==2.19.0
```

Comment:
Wait, how will we sync this?

Reply:
We don't need to sync them at all. Just need this to be higher than 2.18.0 (see https://anaconda.org/conda-forge/dvc-s3 , which has the latest release 2.18.0, because that's when we stopped generating them). So using 2.19.0 as the first version when it got extracted.

Reply:
I understand that it looks a bit odd, but those plugin packages were already released over years in conda, so we kinda have to respect that there are some versions already out. Again, odd to do that specifically for conda, but this is easy and straightforward overall.